Pipfile

```diff
@@ -0,0 +1,30 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+black = "*"
+pytest = "*"
+six = "*"
+codecov = "*"
+"flake8" = "*"
```

Comment:
Why is flake8 in `"` here? Does it matter?

Reply:
I just copied over what already existed in `GatorGrader`. That is how it was but I will check if it needs to be in quotes.